app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb

```diff
@@ -52,7 +52,7 @@ def parse_datacenter(object, props)
     dc_hash = {
       :ems_ref      => object._ref,
       :uid_ems      => object._ref,
-      :type         => "EmsFolder",
+      :type         => "Datacenter",
```

Comment:
Could this change break something on upgrade?  That is, if the database already has a type, I'm not sure you can change it like this.